src/components/card/CommentsTabSidebar.vue

```diff
@@ -0,0 +1,320 @@
+
+import { mapState } from 'vuex'
+import { Avatar } from 'nextcloud-vue'
+import { Actions } from 'nextcloud-vue/dist/Components/Actions'
+import { ActionButton } from 'nextcloud-vue/dist/Components/ActionButton'
+import CommentItem from './CommentItem'
+
+export default {
+	name: 'CommentsTabSidebar',
+	components: {
+		Avatar,
+		Actions,
+		ActionButton,
+		CommentItem,
+		EditorContent
+	},
+	props: {
+		card: {
+			type: Object,
+			default: undefined
+		}
+	},
+	data() {
+		return {
+			newComment: '',
+			isLoading: false,
+			limit: 20,
+			offset: 0,
+
+			editor: new Editor({
+				extensions: [
+					new Mention({
+						// a list of all suggested items
+						items: () => {
+							return this.currentBoard.users
+						},
+						// is called when a suggestion starts
+						onEnter: ({
+							items, query, range, command, virtualNode
+						}) => {
+							this.query = query
+							this.filteredUsers = items
+							this.suggestionRange = range
+							this.renderPopup(virtualNode)
+							// we save the command for inserting a selected mention
+							// this allows us to call it inside of our custom popup
+							// via keyboard navigation and on click
+							this.insertMention = command
+						},
+						// is called when a suggestion has changed
+						onChange: ({
+							items, query, range, virtualNode
+						}) => {
+							this.query = query
+							this.filteredUsers = items
+							this.suggestionRange = range
+							this.navigatedUserIndex = 0
+							this.renderPopup(virtualNode)
+						},
+						// is called when a suggestion is cancelled
+						onExit: () => {
+							// reset all saved values
+							this.query = null
+							this.filteredUsers = []
+							this.suggestionRange = null
+							this.navigatedUserIndex = 0
+							this.destroyPopup()
+						},
+						// is called on every keyDown event while a suggestion is active
+						onKeyDown: ({ event }) => {
+							// pressing up arrow
+							if (event.keyCode === 38) {
+								this.upHandler()
+								return true
+							}
+							// pressing down arrow
+							if (event.keyCode === 40) {
+								this.downHandler()
+								return true
+							}
+							// pressing enter
+							if (event.keyCode === 13) {
+								this.enterHandler()
+								return true
+							}
+							return false
+						},
+						// is called when a suggestion has changed
+						// this function is optional because there is basic filtering built-in
+						// you can overwrite it if you prefer your own filtering
+						// in this example we use fuse.js with support for fuzzy search
+						onFilter: (items, query) => {
+							if (!query) {
+								return items
+							}
+							const fuse = new Fuse(items, {
+								threshold: 0.2,
+								keys: ['uid', 'displayname']
+							})
+							return fuse.search(query)
+						}
+					})
+				],
+				content: '',
+				onUpdate: ({ getHTML }) => {
+					this.newComment = getHTML().replace(/(<p>|<\/p>)/g, '')
```

Comment:
Please make sure all HTML is removed not just p tags and we get the plain text

Reply:
Is there a easy way or just so with replace?

Reply:
```
const element = document.getElementById(getHTML())
this.newComment = element.innerText || element.textContent
```
should do the trick